sdk/storage/storage-file/review/storage-file.api.ts

```diff
@@ -0,0 +1,787 @@
+  getProperties(options?: FileGetPropertiesOptions): Promise<Models.FileGetPropertiesResponse>;
+  getRangeList(options?: FileGetRangeListOptions): Promise<FileGetRangeListResponse>;
+  listHandlesSegment(marker?: string, options?: FileListHandlesSegmentOptions): Promise<Models.FileListHandlesResponse>;
+  resize(length: number, options?: FileResizeOptions): Promise<Models.FileSetHTTPHeadersResponse>;
+  setHTTPHeaders(fileHTTPHeaders?: FileHTTPHeaders, options?: FileSetHTTPHeadersOptions): Promise<Models.FileSetHTTPHeadersResponse>;
+  setMetadata(metadata?: Metadata, options?: FileSetMetadataOptions): Promise<Models.FileSetMetadataResponse>;
+  setProperties(properties?: FileProperties): Promise<SetPropertiesResponse>;
+  // (undocumented)
+  readonly shareName: string;
+  startCopyFromURL(copySource: string, options?: FileStartCopyOptions): Promise<Models.FileStartCopyResponse>;
+  uploadBrowserData(browserData: Blob | ArrayBuffer | ArrayBufferView, options?: UploadToAzureFileOptions): Promise<void>;
+  uploadFile(filePath: string, options?: UploadToAzureFileOptions): Promise<void>;
+  uploadRange(body: HttpRequestBody, offset: number, contentLength: number, options?: FileUploadRangeOptions): Promise<Models.FileUploadRangeResponse>;
+  uploadRangeFromURL(sourceURL: string, sourceOffset: number, destOffset: number, count: number, options?: FileUploadRangeFromURLOptions): Promise<Models.FileUploadRangeFromURLResponse>;
+  uploadResetableStream(streamFactory: (offset: number, count?: number) => NodeJS.ReadableStream, size: number, options?: UploadToAzureFileOptions): Promise<void>;
+  UploadSeekableBlob(blobFactory: (offset: number, size: number) => Blob, size: number, options?: UploadToAzureFileOptions): Promise<void>;
+  uploadStream(stream: Readable, size: number, bufferSize: number, maxBuffers: number, options?: UploadStreamToAzureFileOptions): Promise<void>;
+}
+
+// @public
+interface FileCreateOptions extends FileAndDirectoryCreateCommonOptions, CommonOptions {
+  abortSignal?: AbortSignalLike;
+  fileHTTPHeaders?: FileHTTPHeaders;
+  metadata?: Metadata;
+}
+
+// @public
+interface FileDeleteOptions extends CommonOptions {
+  abortSignal?: AbortSignalLike;
+}
+
+// @public
+interface FileDownloadOptions extends CommonOptions {
+  abortSignal?: AbortSignalLike;
+  maxRetryRequests?: number;
+  progress?: (progress: TransferProgressEvent) => void;
+  rangeGetContentMD5?: boolean;
+}
+
+// @public
+interface FileForceCloseHandlesOptions extends CommonOptions {
+  abortSignal?: AbortSignalLike;
+}
+
+// @public
+interface FileGetPropertiesOptions extends CommonOptions {
+  abortSignal?: AbortSignalLike;
+}
+
+// @public
+interface FileGetRangeListOptions extends CommonOptions {
+  abortSignal?: AbortSignalLike;
+  range?: Range;
+}
+
+// @public
+interface FileListHandlesSegmentOptions extends CommonOptions {
+  abortSignal?: AbortSignalLike;
+  maxresults?: number;
```

Comment:
Does this need to be maxResults with capital R?

Reply:
Yes this should be `maxPageSize` or similar

Reply:
^ name good

Reply:
#5553